src/typeahead/test/typeahead.spec.js

```diff
@@ -842,4 +842,13 @@ describe('typeahead tests', function () {
     expect($scope.select_count).toEqual(1);
   });
 
+  describe('minLength set to 0', function () {
+    it('should open typeahead if input is changed to empty string if defined threshold is 0', function () {
+      var element = prepareInputEl('<div><input ng-model="result" typeahead="item for item in source | filter:$viewValue" typeahead-min-length="0"></div>');
+      changeInputValueTo(element, '');
+
+      expect(element).toBeOpenWithActive(3, 0);
+    });
+  });
```

Comment:
Should it only allow empty `typeahead` to open, if the `input` is focused?

Otherwise several `typeahead` directives  with empty `input`s would simultaneously open at the start, which is probably not intended.